psyneulink/core/components/component.py

```diff
@@ -3196,6 +3196,10 @@ def fill_recursively(arr, value, indices=()):
 
         var = convert_all_elements_to_np_array(copy.deepcopy(var), cast_from=int, cast_to=float)
 
+        # occurs for some Buffer objects
```

Comment:
It'd be more useful to say why it is ok that it occurs and it needs special handling (rather than fix on the Buffer side).